### PR TITLE
New package: Samsung.Gear360ActionDirector version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111988&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=6787887&CDCttType=SW&ModelType=C&ModelName=SM-R210NZWAXAR&VPath=SW/201809/20180917135001161/Gear_360_ActionDirector_180831.exe
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201809/20180917135001161/Gear_360_ActionDirector_180831.exe
```

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung ActionDirector for Gear 360
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml:8]:
```suggestion
PackageName: Gear 360 ActionDirector
```

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Samsung
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml:7]:
```suggestion
Publisher: Cyberlink Corp.
```

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.0.3231
```

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.0.3231
```

File: manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Gear360ActionDirector
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/s/Samsung/Gear360ActionDirector/1.0/Samsung.Gear360ActionDirector.yaml:5]:
```suggestion
PackageVersion: 2.0.0.3231
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image
-Also-
image

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\Gear360ActionDirector\1.0', expected 'manifests\s\Samsung\Gear360ActionDirector\2.0.0.3231'

Is this `1.0` or `2.0.0.3231`?

**Comment by Trenly** [comment]
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\Gear360ActionDirector\1.0', expected 'manifests\s\Samsung\Gear360ActionDirector\2.0.0.3231'
> 
> Is this `1.0` or `2.0.0.3231`?

`2.0.0.3231` is what is written to Control Panel

**Comment by stephengillie** [comment]
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Sa***ung\Gear360ActionDirector\1.0', expected 'manifests\s\Sa***ung\Gear360ActionDirector\2.0.0.3231'

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with: image

Appears to be Interactive Only.

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Pull request closed without merging**